### PR TITLE
involve skywalking agent for linkis services

### PR DESCRIPTION
**Pull request by legendtkl**
ref: https://github.com/apache/incubator-linkis/issues/1330

### CODE REVIEW

**Comment by legendtkl** [comment]
Hi, @peacewong , would you help review this PR?

Thanks, kelu.

**Comment by peacewong** [comment]
> Hi, @peacewong , would you help review this PR?
> 
> Thanks, kelu.

ok，Thanks

File: assembly-combined-package/deploy-config/linkis-env.sh [hunk 1]
```diff
@@ -139,3 +139,6 @@ LINKIS_VERSION=1.0.3
 
 # for install
 LINKIS_PUBLIC_MODULE=lib/linkis-commons/public-module
+
+## If SKYWALKING_AGENT_PATH is set, the Linkis components will be started with Skywalking agent
+SKYWALKING_AGENT_PATH=/appcom/config/skywalking-agent/skywalking-agent.jar
```
Comment (peacewong) [inline assembly-combined-package/deploy-config/linkis-env.sh:144]: Is it better to comment it out by default? It only opens if you need to open SkyWalking

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**